src/enums.h

```diff
@@ -99,4 +105,24 @@ typedef enum FilterType
     PitchShiftFilter
 } FilterType_t;
 
+typedef enum BufferPcmType
```

Comment:
Maybe add a comment about this enum also existing on the Dart side? And add a note about keeping the two in sync?

Reply:
Done.